test/tools/libtesteth/TestHelper.cpp

```diff
@@ -452,6 +456,31 @@ void executeTests(const string& _name, const string& _testPathAppendix, const st
 	}
 }
 
+void addClientInfo(json_spirit::mValue& _v, std::string const& _testSource)
+{
+	for (auto& i: _v.get_obj())
+	{
+		json_spirit::mObject& o = i.second.get_obj();
+		json_spirit::mObject clientinfo;
+
+		string head = executeCmd("git rev-parse HEAD");
+		string comment;
+		if (o.count("_info"))
+		{
+			json_spirit::mObject& existingInfo = o["_info"].get_obj();
+			comment = existingInfo["comment"].get_str();
+		}
+
+		//prepare the relative src path
+		string source = _testSource.substr(_testSource.find("/src/"), _testSource.length());
+
+		clientinfo["filledwith"] = "cpp-ethereum rev-head: " + head;
```

Comment:
Let's have only commit hash as value here, without "cpp-ethereum rev-head", this way it's easier parseble and wastes less disk space.
So something like
```
clientinfo["filledwith_rev"] = head;
```